### PR TITLE
Conversion

### PR DESCRIPTION
**Pull request by PeacefulPieDev**
Finally I have implemented currency conversion, which is using all services that had been done before.

### CODE REVIEW

**Comment by PeacefulPieDev** [comment]
P.S. Although it fixes issues #9 and #10

File: app/src/main/java/com/google/android/gms/samples/vision/ocrreader/OcrCaptureActivity.java [hunk 1]
```diff
@@ -338,9 +340,35 @@ public void onBackPressed() {
...
+		CurrencyCode sourceCurrency = userData.currentCurrency;
+		CurrencyCode targetCurrency = userData.nativeCurrency;
+
+		ICurrencyConverter converter = ServiceFactory.createCurrencyConverter();
+
+		double convertedPrice = 0.0;
+		try {
+			convertedPrice = converter.convert(sourceCurrency, targetCurrency, price);
+		} catch (CurrencyRateFetchingException e) {
+			Snackbar.make(graphicOverlay, "Oh fuck, something is wrong",
```
Comment (oleksandr-bilous) [inline app/src/main/java/com/google/android/gms/samples/vision/ocrreader/OcrCaptureActivity.java]: Not sure if we need to display message like this :rofl:

File: app/src/main/java/currencyConverter/codes/CountryCode.java [hunk 2]
```diff
@@ -1,43 +1,114 @@
...
-	 * @param numeric country code in numeric format
-	 */
-	public CountryCode(int numeric) {
-		this.code = com.neovisionaries.i18n.CountryCode.getByCode(numeric);
-	}
+//
+//	/**
+//	 * Constructs country code from string format
+//	 *
+//	 * @param numeric country code in numeric format
```
Comment (oleksandr-bilous) [inline app/src/main/java/currencyConverter/codes/CountryCode.java]: Do we need this?

File: app/src/main/java/currencyConverter/codes/CurrencyCode.java [hunk 3]
```diff
@@ -1,65 +1,87 @@
...
-        if (o == null || getClass() != o.getClass()) return false;
-        CurrencyCode that = (CurrencyCode) o;
-        return this.code.getNumeric() == that.code.getNumeric();
-    }
+//    @Override
+//    public boolean equals(Object o) {
+//        if (this == o) return true;
+//        if (o == null || getClass() != o.getClass()) return false;
+//        CurrencyCode that = (CurrencyCode) o;
+//        return this.code.getNumeric() == that.code.getNumeric();
```
Comment (oleksandr-bilous) [inline app/src/main/java/currencyConverter/codes/CurrencyCode.java]: -_-

**Review by oleksandr-bilous — APPROVED** [review]
Adjust the code to the following comments and merge. Thx.

**Pull request merged**